resources/schema/spidamin/spidadb/referenced_result.schema

```diff
@@ -4,16 +4,15 @@
     "required": [
         "id",
         "calcResult",
-        "dateModified",
-        "user"
+        "dateModified"
     ],
     "properties": {
         "id": {
             "type": "string",
             "description": "The unique id of this project component"
         },
         "calcResult": {
-            "$ref": "../../spidacalc/result/result.schema"
+            "$ref": "../../spidacalc/results/results.schema"
```

Comment:
is it possible that this could be either a result or results?

Reply:
I don't believe so. We're always putting detailed results at `design.analysisDetails.results[]` even for a single result. The URI in this context was never valid.